### PR TITLE
Fix running as non-root

### PR DESCRIPTION
**Pull request by elprans**
Don't attempt to write outside of `/tmp` when the container isn't
running as root.

### CODE REVIEW

File: edgedb-show-secrets.sh [hunk 1]
```diff
@@ -82,7 +82,7 @@ edb_gs_show_secrets() (
 
   while IFS="=" read -r k v; do
     map["$k"]=$v
-  done < "/etc/edgedb.secrets"
+  done < "/tmp/edgedb.secrets"
```
Comment (fmoor) [inline edgedb-show-secrets.sh:85]: It seems like the dot was changed to a slash in `docker-entryphone-funcs.sh`. Am I reading that correctly? 

```suggestion
  done < "/tmp/edgedb/secrets"
```

**Review by fantix — APPROVED** [review]
LGTM - but when do we need to run without root in Docker?

**Comment by fmoor** [comment]
> LGTM - but when do we need to run without root in Docker?

Heroku is the impetus for this change https://devcenter.heroku.com/articles/container-registry-and-runtime#run-the-image-as-a-non-root-user

**Review by fmoor — APPROVED** [review]

File: edgedb-show-secrets.sh (hunk 1 above)
Comment (elprans) [reply edgedb-show-secrets.sh]: Yeah, it's a typo.

**Pull request merged**